ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/mainnet.yaml

```diff
@@ -153,4 +156,16 @@ BLOB_SIDECAR_SUBNET_COUNT_ELECTRA: 9
 # `uint64(9)`
 MAX_BLOBS_PER_BLOCK_ELECTRA: 9
 # MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK_ELECTRA
-MAX_REQUEST_BLOB_SIDECARS_ELECTRA: 1152
\ No newline at end of file
+MAX_REQUEST_BLOB_SIDECARS_ELECTRA: 1152
+
+# Fulu
+NUMBER_OF_COLUMNS: 128
+NUMBER_OF_CUSTODY_GROUPS: 128
+DATA_COLUMN_SIDECAR_SUBNET_COUNT: 128
+MAX_REQUEST_DATA_COLUMN_SIDECARS: 512
```

Comment:
16384 in spec, see https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L175